python/postprocessing/modules/jme/jetmetUncertainties.py

```diff
@@ -0,0 +1,408 @@
+import ROOT
+import math, os
+import numpy as np
+ROOT.PyConfig.IgnoreCommandLineOptions = True
+
+from PhysicsTools.NanoAODTools.postprocessing.framework.datamodel import Collection, Object
+from PhysicsTools.NanoAODTools.postprocessing.framework.eventloop import Module
+
+#----------------------------------------------------------------------------------------------------
+# CV: global functions copied from PhysicsTools/HeppyCore/python/utils/deltar.py
+
+def deltaPhi(p1, p2):
+    '''Computes delta phi, handling periodic limit conditions.'''
+    res = p1 - p2
+    while res > math.pi:
+        res -= 2*math.pi
+    while res < -math.pi:
+        res += 2*math.pi
+    return res
+
+def deltaR2(e1, p1, e2=None, p2=None):
+    '''Take either 4 arguments (eta,phi, eta,phi) or two particles that have 'eta', 'phi' methods)'''
+    if (e2 == None and p2 == None):
+        return deltaR2(e1.eta, e1.phi, p1.eta, p1.phi)
+    de = e1 - e2
+    dp = deltaPhi(p1, p2)
+    return de*de + dp*dp
```

Comment:
aren't the above functions provided in the tools.py already?
I would import them from there